cuqi/experimental/mcmc/_conjugate.py

```diff
@@ -0,0 +1,50 @@
+import numpy as np
+from cuqi.experimental.mcmc import SamplerNew
+from cuqi.distribution import Posterior, Gaussian, Gamma, GMRF
+from cuqi.implicitprior import RegularizedGaussian, RegularizedGMRF
+
+class ConjugateNew(SamplerNew):
+
```

Comment:
Please add docstring with documentation.

Reply:
Done. Also realized we were missing ConjugateApprox. Now added.